app/src/main/java/io/capibaras/abcall/ui/viewmodels/AccountViewModel.kt

```
@@ -0,0 +1,56 @@
+    init {
+        getUserInfo()
+    }
+
+    private fun getUserInfo() {
+        if (stateMediator.isLoading) return
+        stateMediator.setLoadingState(true)
+        viewModelScope.launch {
+            try {
+                val response: User = usersRepository.getUserInfo()
+                user = response
+            } catch (e: IOException) {
```

detekt

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)